crates/cheatnet/src/runtime_extensions/forge_runtime_extension/mod.rs

```diff
@@ -708,25 +708,37 @@ fn concat_u128_bytes(low: &[u8; 32], high: &[u8; 32]) -> [u8; 32] {
 }
 
 #[must_use]
-pub fn get_all_execution_resources(runtime: ForgeRuntime) -> UsedResources {
+pub fn update_test_execution_resources_and_get_them(runtime: ForgeRuntime) -> UsedResources {
```

Comment:
I think that's a clear sign that this should be 2 functions and not 1 (function should do only one thing)

Reply:
Couldn't do it before due to ownership but now I can